style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  box-sizing: border-box;
+  font-family: sans-serif;
+  margin: 0rem;
+}
+
+nav {
+  font-weight: bold;
+  text-decoration: none;
+}
+
+a, a:visited, a:active {
+  text-decoration: none;
+  color: white;
+  padding: 0.5rem;
+}
+
+p {
+  margin: 0.5rem;
+}
+
+.top-nav {
+  width: 100%;
+  border: solid 1px #006064;
+  background: #006064;
+  overflow: auto;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+}
+
+.nav-list_item {
+  margin: 0.5rem 0.5rem;
+  text-transform: uppercase;
+  background: gray;
+  padding: 0.5rem;
+  border-radius: 5px;
+  text-decoration: none;
+}
+
+.nav-list_item--brand {
+  flex: 1;
+  text-align: center;
+  text-transform: uppercase;
+  padding: 0.5rem;
+}
+
+.nav-list_item:hover {
+  background: lightgrey;
+}
+
+.body-img {
+  display: inline-block;
+  margin: 1rem 0 1rem 0;
+}
+
+.row {
+  margin: auto;
+  overflow: auto;
+}
+
+.column {
+  width: calc(100% / 3 - 1rem - 1rem);
+  margin: 1rem;
+  min-width: 15rem;
+  text-align: center;
+  border: 3px solid pink;
+  border-radius: 5px;
+  box-shadow: 5px 5px 3px #888888;
+  float: left;
+}
+
+img {
+  width: 100%;
+}
+
+.card-text-content {
+  text-align: center;
+  display: inline-block;
```

Comment:
By setting `inline-block` here, the width of the text decides the content size, so the first element is limited in size by the text size. By leaving this as `block`, then the default width will be 100%. If you want extra room between that and the edge of the card, use some of the box model properties.